tests/check_add_email.py

```diff
@@ -14,44 +15,36 @@
 @pytest.mark.nondestructive
 class TestAddEmail(BaseTest):
 
-    @pytest.mark.travis
-    def test_add_email(self, mozwebqa):
-        user = self.create_verified_user(mozwebqa.selenium, mozwebqa.timeout)
-        user.additional_emails.append('%s_1@restmail.net' % user.id)
-
-        mozwebqa.selenium.get('%s/' % mozwebqa.base_url)
-        self.log_out(mozwebqa.selenium, mozwebqa.timeout)
-        mozwebqa.selenium.find_element(*self._persona_login_button_locator).click()
+    def test_add_email(self, base_url, selenium, timeout, verified_user, new_email):
+        browser_id = BrowserID(selenium, timeout)
+        browser_id.sign_in(verified_user['email'], verified_user['password'])
+        WebDriverWait(selenium, timeout).until(lambda s: s.find_element(
+            *self._persona_logged_in_indicator_locator).is_displayed())
+        self.log_out(selenium, timeout)
+        selenium.find_element(*self._persona_login_button_locator).click()
 
         from browserid.pages.sign_in import SignIn
-        signin = SignIn(mozwebqa.selenium, mozwebqa.timeout)
+        signin = SignIn(selenium, timeout)
         signin.click_add_another_email_address()
-        signin.new_email = user.additional_emails[0]
-        assert signin.new_email == user.additional_emails[0], "new email getter failed"
+        signin.new_email = new_email
+        assert signin.new_email == new_email, 'new email getter failed'
```

Comment:
I believe so. These tests are for the page object, so it's important that setting the value was successful.